examples/auth/oauth2/static-route/Makefile

```diff
@@ -0,0 +1,6 @@
+.PHONY: all
+all:
+	-kubectl delete -f ./manifests.yaml
+	-kubectl delete -f ./api.yaml
+	-kubectl apply -f ./manifests.yaml
+	-kubectl apply -f ./api.yaml
```

Comment:
Do we need a makefile for this? is this common that we will need to do this?

Reply:
I'll remove  the `Makefile`.